api/server/server.go

```diff
@@ -149,10 +152,8 @@ func httpError(w http.ResponseWriter, err error) {
 		statusCode = http.StatusForbidden
 	}
 
-	if err != nil {
-		logrus.Errorf("HTTP Error: statusCode=%d %v", statusCode, err)
-		http.Error(w, err.Error(), statusCode)
-	}
+	logrus.Errorf("HTTP Error: statusCode=%d %v", statusCode, err)
```

Comment:
while you're at it, you can start using the preferred logrus fields format from #11554:

`logrus.WithFields(logrus.Fields{"statusCode": statusCode, "err": err}).Error("HTTP Error")`